private-templates-service/client/src/components/Dashboard/PreviewModal/TemplateInfo/TemplateInfo.tsx

```diff
@@ -105,30 +105,37 @@ class TemplateInfo extends React.Component<Props, State> {
     const {
       isLive,
       tags,
-      createdAt, 
+      createdAt,
       instances,
     } = this.props.template;
     const { onClose } = this.props;
 
+    let createdAtParsed = "";
+
+    if (createdAt) {
```

Comment:
personal preference not to make if checks against non-Boolean types, it's a nit, though